### PR TITLE
Add pmysql-dsb support with tests

### PR DESCRIPTION
**Pull request by vjtirumalai**
- [x] Local build passes
- [x] Functional tests passed with 0.23 
- [x] Works with CF deployer and shpanCRB 0.24

Defer to next iteration:
- [ ] Make functional tests work with CRB FS version based on 0.24
- [ ] CI pending

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- This is our default parent pom, it is useful to keep general versions
+	 of artifacts, plugins and maven repository definitions -->
+    <parent>
```
Comment (shpandrak) [inline pom.xml]: There is no need for this anymore, lets talk

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (shpandrak) [inline pom.xml:1]: Please remove this entire pom and flatten the project to have only the mysql-dsb as root pom. this pom adds nothing.. lets talk

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    <modules>
+        <module>mysqldsb</module>
+    </modules>
+
+
+    <dependencyManagement>
+        <dependencies>
+
+            <dependency>
+                <groupId>com.emc.ocopea.utilities</groupId>
```
Comment (shpandrak) [inline pom.xml]: the groupId of utilities is incorrect. please use:
    <groupId>com.emc.ocopea.util</groupId>

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    <groupId>com.emc.ocopea.mysql-dsb</groupId>
+    <packaging>pom</packaging>
+    <artifactId>dsb-super-parent</artifactId>
+    <name>Parent project for mysql-dsb</name>
+    <version>0.1</version>
+
+    <properties>
+        <microservice.api.version>1.0</microservice.api.version>
+        <microservice.runtime.version>0.1</microservice.runtime.version>
+        <ocopea.dsb.version>0.38</ocopea.dsb.version>
```
Comment (shpandrak) [inline pom.xml]: lets talk about component version and how we manage them, I want it to be the same across the projects

File: mysqldsb/pom.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (shpandrak) [inline mysqldsb/pom.xml]: This pom should be the projects root pom..

File: mysqldsb/pom.xml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.emc.ocopea.mysql-dsb</groupId>
+        <artifactId>dsb-super-parent</artifactId>
+        <version>0.1</version>
```
Comment (shpandrak) [inline mysqldsb/pom.xml]: version should be snapshot, but lets talk...

File: mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+import org.slf4j.LoggerFactory;
+
+/**
+ * Created by liebea on 1/16/17.
+ * Drink responsibly
+ */
+public class MySqlDSBMicroService extends MicroService {
+    private static final String SERVICE_NAME = "MySql DSB";
+    private static final String SERVICE_BASE_URI = "mysql-dsb";
+    private static final String SERVICE_DESCRIPTION = "MySql DSB implementation in java";
```
Comment (shpandrak) [inline mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java]: This appears in the API, no need to mention it is in java :) should say something about PCF of p-mysql

File: mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java [hunk 8]
```diff
@@ -0,0 +1,48 @@
+// Copyright (c) [2017] Dell Inc. or its subsidiaries. All Rights Reserved.
```
Comment (shpandrak) [inline mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java]: The package directory structure does not match the package name (there is nazgul in the folder name...)

File: pom.xml (hunk 2 above)
Comment (vjtirumalai) [reply pom.xml:1]: Thanks for fixing

**Comment by vjtirumalai** [comment]
Will wait for CRB FS side changes before this gets merged.

File: mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java (hunk 8 above)
Comment (vjtirumalai) [reply mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java]: Thanks for fixing.

File: pom.xml (hunk 2 above)
Comment (shpandrak) [reply pom.xml:1]: At your service

File: pom.xml (hunk 1 above)
Comment (vjtirumalai) [reply pom.xml]: Thanks for fixing.

File: pom.xml (hunk 3 above)
Comment (vjtirumalai) [reply pom.xml]: Fixed by removing.

File: pom.xml (hunk 4 above)
Comment (vjtirumalai) [reply pom.xml]: sure next iteration.

File: mysqldsb/pom.xml (hunk 5 above)
Comment (vjtirumalai) [reply mysqldsb/pom.xml]: Thanks for fixing

File: mysqldsb/pom.xml (hunk 6 above)
Comment (vjtirumalai) [reply mysqldsb/pom.xml]: Thanks for fixing.

File: mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java (hunk 7 above)
Comment (vjtirumalai) [reply mysqldsb/service/src/main/java/com/emc/nazgul/dsb/mysql/MySqlDSBMicroService.java]: Good point. Fixed.

**Review by shpandrak — APPROVED** [review]

**Pull request merged**